File: Movie.cs
class Movie
{
    string title;
    private List<MovieScreening> _screenings;


    public Movie(string title)
    {
        this.title = title;
        _screenings = new List<MovieScreening>();
    }

    static void Addscreening(MovieScreening screening)
    {
        _screenings.Add(screening);
    }

    public override string ToString()
    {
        return title;
    }
}

File: Movie.java
import java.util.ArrayList;
import java.util.List;

public class Movie {
    private String title;
    private List<MovieScreening> screenings;

    public Movie(String title) {
        this.title = title;
        this.screenings = new ArrayList<>();
    }

    public void addScreening(MovieScreening screening) {
        screenings.add(screening);
    }

    @Override
    public String toString() {
        return "Movie Title: " + title;
    }
}